README.md

```diff
@@ -204,6 +204,7 @@ const main = () => {
 - `visualizeAttributes` (optional)(`string[]`): List of attribute names (strings) which should be visualized. Defaults to the first 3 if no value is provided. If an empty list is provided, displays no attributes.
 - `visualizeUpsetAttributes` (optional)(`boolean`): Whether or not to visualize UpSet generated attributes (`degree` and `deviation`). Defaults to `false`.
 - `allowAttributeRemoval` (optional)(`boolean`): Whether or not to allow the user to remove attribute columns. This should be enabled only if there is an option within the parent application which allows for attributes to be added after removal. Default attribute removal behavior in UpSet 2.0 is done via context menu on attribute headers. Defaults to `false`.
+- `userEditPerms` (optional)(`boolean`): Whether or not the user has plot information edit permissions.
```

Comment:
I recommend `userCanEdit` or even `userCanEditPlotInfo` instead, but this is fine if you prefer. And I think `Whether or not the user can edit the plot information in the text descriptions sidebar` is a clearer description, but again, this is personal preference.

Reply:
Resolved in https://github.com/visdesignlab/upset2/pull/415/commits/9ecb9eb6f1347bc338aa81b224dcddee605c048f

I agree with your recommendations here.